app/assets/stylesheets/components/_nav.scss

```diff
@@ -46,60 +46,68 @@
   }
 }
 
-.nav-dropdown {
+.nav {
   font-family: $header-font;
-  padding: 0 rem-calc(30);
-  margin-right: rem-calc(30);
-  
-  display: flex;
-  flex-direction: column;
-  position: absolute;
-  top: 0;
-  right: 0;
-  z-index: 100;
-  
-  &:hover, &:active {
-    background-color: $green;
-  }
+  background-color: $green;
+  color: $black;
+  list-style: none;
 
   a {
     text-decoration: none;
   }
 
-  img {
-    margin-top: rem-calc(3);
-    width: rem-calc(34); height: rem-calc(34);
-  }
+  &-dropdown {
+    padding: 0 rem-calc(30);
+    margin-right: rem-calc(30);
 
-  &__list {
-    background-color: $green;
-    margin-top: rem-calc(44);
-    padding: rem-calc(20) 0;
-    min-width: rem-calc(160);
-    
+    display: flex;
+    flex-direction: column;
     position: absolute;
+    top: 0;
     right: 0;
-    z-index: 50;
-    
-    li {
-      color: $black;
-      list-style: none;
-      padding: 0 rem-calc(20);
+    z-index: 100;
+
+    &:hover,
+    &:active {
+      background-color: $green;
     }
 
-    li:hover {
-      background-color: $green-light;
+    img {
+      margin-top: rem-calc(3);
+      width: rem-calc(34);
+      height: rem-calc(34);
+    }
+
+    &__list {
```

Comment:
Spacing is a little off on this. In the design, min-width is 196px, there's 29px padding on the left, 21px padding at the top and about 16px space between the items (though the line height is larger than the text height, so it's hard to be precise)! And in the design, the logout link isn't spaced the same as the top two, but I'd probably just base it off the top two

<img width="246" alt="Screenshot 2022-01-31 at 10 48 03" src="https://user-images.githubusercontent.com/40244233/151780555-4635c230-b944-42e5-a098-b2e4a970e2df.png">